tsconfig.base.json

```
@@ -56,6 +57,8 @@
     "stripInternal": true,
     "suppressExcessPropertyErrors": false,
     "suppressImplicitAnyIndexErrors": false,
-    "target": "es6"
+    "target": "es6",
+    "typeRoots": [
+      "raw.d.ts",]
```

Why do you use this ?
```suggestion
    "typeRoots": ["raw.d.ts"]
```

---

I don't need it anymore. I used to add 'raw' to the svg path for the import to succed.
raw.d.ts is now deleted

---

I'm curious too of why do we need this ? Copilot says it should be a folder and not a file here, how does it work ?

---

it was a try to import the `.svg` files with `?raw` suffix, but we still add an error, it should be removed now

---

done